packages/commerce/frontend/src/hooks/use-toast.tsx

```diff
@@ -0,0 +1,183 @@
+import { useEffect, useState } from 'react'
+import type { ToastActionElement, ToastProps } from '@/components/ui/toast.tsx'
+
+const TOAST_LIMIT = 1
+const TOAST_REMOVE_DELAY_MS = 3_000
```

Comment:
What is this value used for?

Reply:
After how much time the toast should be dismissed.
https://github.com/interledger/testnet/blob/0a2fea68c7b7509b6a4ce3f29c166163706282c9/packages/commerce/frontend/src/hooks/use-toast.tsx#L59-L65